tests/foreman/api/test_contentviewversion.py

```diff
@@ -163,8 +163,15 @@ def test_incremental_update_puppet(self):
         ).create()
         with open(get_data_file(PUPPET_MODULE_NTP_PUPPETLABS), 'rb') as handle:
             puppet_repo.upload_content(files={'content': handle})
+        # Extract all the available puppet modules.
+        puppet_modules = content_view.available_puppet_modules()['results']
```

Comment:
The variable `puppet_modules` was added only to make the next 2 lines of code short enough and avoid pylint yelling at us.